garden_ai/app/create.py

```diff
@@ -0,0 +1,171 @@
+        ...,
+        "-t",
+        "--title",
+        prompt="Please enter a title for your Garden",
+        help="Provide an official title (as it should appear in citations)",
+        rich_help_panel="Required",
+    ),
+    year: str = typer.Option(
+        str(datetime.now().year),  # default to current year
+        "-y",
+        "--year",
+        rich_help_panel="Required",
+    ),
+    contributors: List[str] = typer.Option(
+        None,
+        "-c",
+        "--contributor",
+        help=(
+            "Acknowledge a contributor in this Garden. Repeat to indicate multiple (like --author). "
+        ),
+        rich_help_panel="Recommended",
+    ),
+    description: Optional[str] = typer.Option(
+        None,
+        "-d",
+        "--description",
+        help=(
+            "A brief summary of the Garden and/or its purpose, to aid discovery by other Gardeners."
+        ),
+        rich_help_panel="Recommended",
+    ),
+    tags: List[str] = typer.Option(
+        None,
+        "--tag",
+        help="Add a tag, keyword, key phrase or other classification pertaining to the Garden.",
+        rich_help_panel="Recommended",
+    ),
+    verbose: bool = typer.Option(
+        False, help="If true, pretty-print Garden's metadata when created."
+    ),
+):
+    """Create a new Garden"""
+    while not authors:
+        # repeatedly prompt for at least one author until one is given
+        name = validate_name(Prompt.ask("Please enter at least one author (required)"))
+        if not name:
+            continue
+
+        authors = [name]
+        # prompt for additional authors until one is *not* given
+        while True:
+            name = validate_name(
+                Prompt.ask("Add another author? (leave blank to finish)")
+            )
+            if name:
+                authors += [name]
+            else:
+                break
+
+    if not contributors:
+        name = validate_name(
+            Prompt.ask("Acknowledge a contributor? (leave blank to skip)")
+        )
+        if name:
+            contributors = [name]
+            while True:
+                name = validate_name(
+                    Prompt.ask("Add another contributor? (leave blank to finish)")
+                )
+                if name:
+                    authors += [name]
+                else:
+                    break
+
+    if not description:
+        description = Prompt.ask(
+            "Provide a brief description of this Garden, to aid in discovery (leave blank to skip)"
+        )
+
+    client = GardenClient()
+
+    garden = client.create_garden(
+        authors=authors,
+        title=title,
+        year=year,
+        description=description,
+        contributors=contributors,
+        tags=tags,
+    )
+
+    client.register_metadata(garden, out_dir=LOCAL_STORAGE / "gardens")
+
+    if verbose:
+        with open(LOCAL_STORAGE / "gardens" / f"{garden.garden_id}.json", "r") as f_in:
```

Comment:
For follow-up work: I'd like to push us to just one local database.json file with all the state. Less futzing around with the filesystem, easier to whip up fixtures for testing.